vector/src/main/java/im/vector/services/EventStreamService.java

```diff
@@ -853,10 +854,10 @@ public void refreshStatusNotification() {
         if (mNotificationState == NotificationState.NONE) {
             Notification notification = NotificationUtils.buildMessageNotification(getApplicationContext(), true);
 
+            final NotificationManagerCompat nm = NotificationManagerCompat.from(EventStreamService.this);
             if (null != notification) {
                 mNotificationState = NotificationState.DISPLAYING_EVENTS_NOTIFICATIONS;
-                startForeground(NOTIFICATION_ID, notification);
-                mIsForeground = true;
+                nm.notify(NOTIFICATION_ID, notification);
```

Comment:
This notification is used to keep alive the application alive during the application launch. (it should be updated after the first sync request).

It is used by the Google play AND the  Fdroid versions.

Without this foreground notifications, the application might be killed during its launch.
for the google play version, some notifications might not be triggered.
for the fdroid version, the application might be killed whereas it was starting on boot.

Reply:
Setting this to a foreground notification causes the first notification when starting the app to become unswipable on both google play and fdroid. However, I don't want to do too big of a restructure, so I'll just restore the old behavior.